src/main/resources/webroot/keys/sharedid-public-key

```diff
@@ -0,0 +1,6 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
i'd rename this location to `keys/sharedid.pub` since that's the convention for RSA keys.